test/azuremetadata/generated/Lro/README.md

```diff
@@ -30,16 +26,17 @@ npm install @azure/ms-rest-nodeauth@"^3.0.0"
 
 ##### Sample code
 
+While the below sample uses the interactive login, other authentication options can be found in the [README.md file of @azure/ms-rest-nodeauth](https://www.npmjs.com/package/@azure/ms-rest-nodeauth) package
 ```typescript
-import * as msRest from "@azure/ms-rest-js";
+const msRest = require("@azure/ms-rest-js")
 import * as msRestAzure from "@azure/ms-rest-azure-js";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { AutoRestLongRunningOperationTestService, AutoRestLongRunningOperationTestServiceModels, AutoRestLongRunningOperationTestServiceMappers } from "";
+const msRestNodeAuth = require("@azure/ms-rest-nodeauth")
+const { AutoRestLongRunningOperationTestService } = require("")

```

Comment:
This empty string is not caused by the changes in this PR and seems to be misconfiguration of the test generation script + test swagger.